### PR TITLE
Add coloring to fx graph print out

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #128476

Note: Won't land immediately, at least I'll need to add a color option to the field. But curious if any tests fail.

Old:
image

New:
image

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

**Comment by ezyang** [comment]
Oh yeah... we test on the output strings here...... I'm not sure what to do 🤣

**Comment by Chillee** [comment]
@ezyang just add a flag lol

**Comment by ezyang** [comment]
Good luck 👹

File: torch/fx/graph_module.py [hunk 1]
```diff
@@ -821,12 +821,12 @@ def __copy__(self):
...
 
     @compatibility(is_backward_compatible=False)
-    def print_readable(self, print_output=True, include_stride=False, include_device=False):
+    def print_readable(self, print_output=True, include_stride=False, include_device=False, colored=False):
         """
         Return the Python code generated for current GraphModule and its children GraphModules
         """
         verbose_python_code = self._graph.python_code(
-            root_module="self", verbose=True, include_stride=include_stride, include_device=include_device
+            root_module="self", verbose=True, include_stride=include_stride, include_device=include_device, colored=colored
```
Comment (ezyang) [inline torch/fx/graph_module.py:829]: nit: consider perhaps `ansi_color`, to make it clear that it's ANSI control codes color? Not a big deal either way

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**